packages/cf-component-wizard/README.md

```diff
@@ -0,0 +1,99 @@
+# cf-component-wizard
+
+> CloudFlare Wizard Component
+
+## Installation
+
+```sh
+$ npm install cf-component-wizard
+```
+
+## Usage
+
+```js
+const React = require('react');
+const {render} = require('react-dom');
+const Wizard = require('../../src/index');
+const {Button} = require('cf-component-button');
+
+const {
+  GatewayDest,
+  GatewayProvider
+} = require('react-gateway');
+
+class Component extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeStep: 'step1',
```

Comment:
Will this ever be something other than a numerical value? Should we have this just as a integer so that something like this would be possible?

```js
this.setState((state) => {
  return {activeStep: state.activeStep + 1}
});
```